### PR TITLE
fix more reserved keywords identifier issues

### PR DESCRIPTION
**Pull request by azjezz**

### CODE REVIEW

File: tests/third_party_tests.rs [hunk 1]
```diff
@@ -118,42 +135,64 @@ fn phabel() {
             // Uses non-standard async/await syntax
             "tests/TargetFuture/AwaitTest.php",
         ],
+        // fails to install sometimes..
+        false,
```
Comment (azjezz) [inline tests/third_party_tests.rs:139]: fails on my laptop, not sure why lol

Comment (danog) [reply tests/third_party_tests.rs:139]: Hmm, what's the error?

Comment (azjezz) [reply tests/third_party_tests.rs:139]:
```
PHP Fatal error:  Uncaught TypeError: array_merge(): Argument #1 must be of type array, bool given in phar:///usr/local/bin/composer.phar/src/Composer/Config.php:191
Stack trace:
#0 phar:///usr/local/bin/composer.phar/src/Composer/Config.php(191): array_merge(true, Array, true)
#1 phar:///usr/local/bin/composer.phar/src/Composer/Factory.php(350): Composer\Config->merge(Array, './composer.json')
#2 phar:///usr/local/bin/composer.phar/src/Composer/Factory.php(643): Composer\Factory->createComposer(Object(Composer\IO\ConsoleIO), Array, false, '/home/azjezz/Pr...', true, false)
#3 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(444): Composer\Factory::create(Object(Composer\IO\ConsoleIO), NULL, false, false)
#4 phar:///usr/local/bin/composer.phar/src/Composer/Command/BaseCommand.php(65): Composer\Console\Application->getComposer(false, false, false)
#5 phar:///usr/local/bin/composer.phar/src/Composer/Command/BaseCommand.php(150): Composer\Command\BaseCommand->getComposer(false, false, false)
#6 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Command/Command.php(211): Composer\Command\BaseCommand->initialize(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(835): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(185): Symfony\Component\Console\Application->doRunCommand(Object(Composer\Command\InstallCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(335): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(117): Composer\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(130): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 phar:///usr/local/bin/composer.phar/bin/composer(73): Composer\Console\Application->run()
#13 /usr/local/bin/composer.phar(29): require('phar:///usr/loc...')
#14 {main}
  thrown in phar:///usr/local/bin/composer.phar/src/Composer/Config.php on line 191

Fatal error: Uncaught TypeError: array_merge(): Argument #1 must be of type array, bool given in phar:///usr/local/bin/composer.phar/src/Composer/Config.php:191
Stack trace:
#0 phar:///usr/local/bin/composer.phar/src/Composer/Config.php(191): array_merge(true, Array, true)
#1 phar:///usr/local/bin/composer.phar/src/Composer/Factory.php(350): Composer\Config->merge(Array, './composer.json')
#2 phar:///usr/local/bin/composer.phar/src/Composer/Factory.php(643): Composer\Factory->createComposer(Object(Composer\IO\ConsoleIO), Array, false, '/home/azjezz/Pr...', true, false)
#3 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(444): Composer\Factory::create(Object(Composer\IO\ConsoleIO), NULL, false, false)
#4 phar:///usr/local/bin/composer.phar/src/Composer/Command/BaseCommand.php(65): Composer\Console\Application->getComposer(false, false, false)
#5 phar:///usr/local/bin/composer.phar/src/Composer/Command/BaseCommand.php(150): Composer\Command\BaseCommand->getComposer(false, false, false)
#6 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Command/Command.php(211): Composer\Command\BaseCommand->initialize(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(835): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(185): Symfony\Component\Console\Application->doRunCommand(Object(Composer\Command\InstallCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(335): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(117): Composer\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(130): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 phar:///usr/local/bin/composer.phar/bin/composer(73): Composer\Console\Application->run()
#13 /usr/local/bin/composer.phar(29): require('phar:///usr/loc...')
#14 {main}
  thrown in phar:///usr/local/bin/composer.phar/src/Composer/Config.php on line 191
```

i don't have the energy to figure out why rn ( php 8.1.1 / composer 2.2.0 )

Comment (azjezz) [reply tests/third_party_tests.rs:139]:
```
PHP Fatal error:  Uncaught TypeError: array_merge(): Argument #1 must be of type array, bool given in phar:///usr/local/bin/composer.phar/src/Composer/Config.php:191
Stack trace:
#0 phar:///usr/local/bin/composer.phar/src/Composer/Config.php(191): array_merge(true, Array, true)
#1 phar:///usr/local/bin/composer.phar/src/Composer/Factory.php(350): Composer\Config->merge(Array, './composer.json')
#2 phar:///usr/local/bin/composer.phar/src/Composer/Factory.php(643): Composer\Factory->createComposer(Object(Composer\IO\ConsoleIO), Array, false, '/home/azjezz/Pr...', true, false)
#3 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(444): Composer\Factory::create(Object(Composer\IO\ConsoleIO), NULL, false, false)
#4 phar:///usr/local/bin/composer.phar/src/Composer/Command/BaseCommand.php(65): Composer\Console\Application->getComposer(false, false, false)
#5 phar:///usr/local/bin/composer.phar/src/Composer/Command/BaseCommand.php(150): Composer\Command\BaseCommand->getComposer(false, false, false)
#6 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Command/Command.php(211): Composer\Command\BaseCommand->initialize(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(835): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(185): Symfony\Component\Console\Application->doRunCommand(Object(Composer\Command\InstallCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(335): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(117): Composer\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(130): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 phar:///usr/local/bin/composer.phar/bin/composer(73): Composer\Console\Application->run()
#13 /usr/local/bin/composer.phar(29): require('phar:///usr/loc...')
#14 {main}
  thrown in phar:///usr/local/bin/composer.phar/src/Composer/Config.php on line 191

Fatal error: Uncaught TypeError: array_merge(): Argument #1 must be of type array, bool given in phar:///usr/local/bin/composer.phar/src/Composer/Config.php:191
Stack trace:
#0 phar:///usr/local/bin/composer.phar/src/Composer/Config.php(191): array_merge(true, Array, true)
#1 phar:///usr/local/bin/composer.phar/src/Composer/Factory.php(350): Composer\Config->merge(Array, './composer.json')
#2 phar:///usr/local/bin/composer.phar/src/Composer/Factory.php(643): Composer\Factory->createComposer(Object(Composer\IO\ConsoleIO), Array, false, '/home/azjezz/Pr...', true, false)
#3 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(444): Composer\Factory::create(Object(Composer\IO\ConsoleIO), NULL, false, false)
#4 phar:///usr/local/bin/composer.phar/src/Composer/Command/BaseCommand.php(65): Composer\Console\Application->getComposer(false, false, false)
#5 phar:///usr/local/bin/composer.phar/src/Composer/Command/BaseCommand.php(150): Composer\Command\BaseCommand->getComposer(false, false, false)
#6 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Command/Command.php(211): Composer\Command\BaseCommand->initialize(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(835): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(185): Symfony\Component\Console\Application->doRunCommand(Object(Composer\Command\InstallCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(335): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php(117): Composer\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php(130): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 phar:///usr/local/bin/composer.phar/bin/composer(73): Composer\Console\Application->run()
#13 /usr/local/bin/composer.phar(29): require('phar:///usr/loc...')
#14 {main}
  thrown in phar:///usr/local/bin/composer.phar/src/Composer/Config.php on line 191
```

i don't have the energy to figure out why rn ( php 8.1.1 / composer 2.2.0 )

Comment (azjezz) [reply tests/third_party_tests.rs:139]: nvm, updating to composer 2.4.4 solved it

Comment (danog) [reply tests/third_party_tests.rs:139]: Try running composer self-update to upgrade to 2.4.4, seems just a composer bug

**Pull request merged**